src/main.rs

```diff
@@ -166,7 +166,7 @@ fn cli() -> Command {
         ))
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Was there a particular reason to change this? I feel like I might have asked this previously if I've seen this change, but I don't understand why we would want to try to coerce tokio into executing this code on only a single thread. It's usually pretty intelligent about deciding how to handle parallelization under the hood.

Reply:
Fraser found out that the client was not multi thread and removed "rt-multi-thread feature flag" on dev

quoting him 
```
I have no idea why I decided on the multi-threaded runtime for tokio.  There's no need for it in the client
```

https://github.com/casper-ecosystem/casper-client-rs/pull/153/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L52

``` tokio can efficiently handle many concurrent I/O operations even on a single thread due to its asynchronous nature.```

I believe until the client does not make simultaneous requests or many while waiting for RPC responses the client does not need this dependency.